FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -619,4 +625,57 @@ - (FIRConfigUpdateListenerRegistration *)addOnConfigUpdateListener:
   return [self->_configRealtime addConfigUpdateListener:listener];
 }
 
+#pragma mark - Rollout
+
+- (void)addRemoteConfigInteropSubscriber:(id<FIRRolloutsStateSubscriber>)subscriber {
+  [[NSNotificationCenter defaultCenter]
+      addObserverForName:RolloutsStateDidChangeNotificationName
+                  object:self
+                   queue:nil
+              usingBlock:^(NSNotification *_Nonnull notification) {
+                FIRRolloutsState *rolloutsState =
+                    notification.userInfo[RolloutsStateDidChangeNotificationName];
+                [subscriber rolloutsStateDidChange:rolloutsState];
+              }];
+  // Send active rollout metadata stored in persistence while app launched if there is activeConfig
+  NSString *FQNamespace = [self fullyQualifiedNamespace:_FIRNamespace];
```

Comment:
nit: would expand the variable name to `fullyQualifiedNamespace`

Reply:
Fixed